src/RubberDuckyParking/parking_api/migrations/0001_initial.py

```diff
@@ -127,7 +135,7 @@ class Migration(migrations.Migration):
                 ('hash_str', models.CharField(max_length=100)),
                 ('location', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='parking_api.location')),
                 ('spot', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='parking_api.parkingspot')),
-                ('user', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='parking_api.baseuser')),
+                ('user', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
```

Comment:
I don't understand the parameter "to" or how exactly django user auth model works, but this looks good.